tests/fieldio/ex_io.cpp

```diff
@@ -24,62 +25,63 @@
 // Test reading EX file containing both a data and node group of same name.
 TEST(FieldIO, data_and_node_group)
 {
-	ZincTestSetupCpp zinc;
```

Comment:
Please undo tabs to spaces on the unmodified lines in this file... diff is impossible.